pkg/ingester/ingester_test.go

```diff
@@ -424,7 +424,7 @@ func TestIngesterSpreadFlush(t *testing.T) {
 	_, _ = pushTestSamples(t, ing, 4, 1, int(cfg.MaxChunkAge.Seconds()-1)*1000)
 
 	// wait beyond flush time so first set of samples should be sent to store
-	time.Sleep(cfg.FlushCheckPeriod * 2)
+	time.Sleep(cfg.FlushCheckPeriod * 4)
```

Comment:
I don't quite understand why this was needed.

Reply:
Because CI was failing; not all chunks had been flushed by the time it checked.  I could understand that this PR makes the flush take longer, though the queue should clear within one flush period.  On my laptop the failure was intermittent; after several hours of probing I found that Go timers just aren't that reliable - I would see the 20ms timer fire much later, sometimes by 100ms (and if that happens the test still fails after this change).  Perhaps related to https://github.com/golang/go/issues/38860 though I can't see what would make any goroutines CPU-bound at the point we do this Sleep().

I could take that change back out and see how we get on.

Reply:
> I could take that change back out and see how we get on.

No need for me. Thanks for explanation.